Adapt documentation to terminal naming changes (#76)
uyuni-project/uyuni-docs: modules/retail/pages/retail-terminal-names.adoc

----

The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.

> You can configure this value in the xref:salt:formula-branch-network.adoc[] settings for the branch server.

Reviewer: ```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

> You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[].

Reviewer: ```suggestion
You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

The [systemitem]``Manufacturer``, [systemitem]``ProductName``, and [systemitem]``SerialNumber`` are provided by the terminal hardware BIOS.
If the terminal does not provide a serial number, it will be omitted from the terminal name.

The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.
Added unique ID is a requirement for successful terminal deployment. Without unique ID, subsequent terminal registrations will fail.

=== Naming by Hostname

Terminal names that are derived from the hostname use this format:

----
BranchID.Hostname-UniqueID
----

For example:

----
B002.terminal-c643
----

The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.

> You can configure this value in the xref:salt:formula-branch-network.adoc[] settings for the branch server.

Reviewer: ```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[].

The [systemitem]``Hostname`` is the plain hostname (without domain part) of the terminal.

The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.

> You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branch-network.adoc[].

Reviewer: ```suggestion
You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

> === Naming using the FQDN naming scheme

Reviewer: ```suggestion
=== Naming by ``FQDN``
```

> Terminal names that are derived from the Fully Qualified Domain Names  use this format:

Reviewer: ```suggestion
Terminal names that are derived from the Fully Qualified Domain Names (FQDN) use this format:
```

The [systemitem]``BranchID`` is the unique identifier for the branch server that the terminal is connected to.

> You can configure this value in the xref:salt:formula-branch-network.adoc[Branch Network formula] settings for the branch server.

Reviewer: ```suggestion
You can configure this value in the xref:salt:formula-branchnetwork.adoc[] settings for the branch server.
```

> You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branch-network.adoc[Branch Network formula].

Reviewer: ```suggestion
You can disable this prefix by toggling the [systemitem]``Do not prefix salt client ID with Branch ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```

The [systemitem]``FQDN`` is the fully-qualified domain name of the terminal.

The [systemitem]``UniqueID`` is the first four characters of a generated machine identification number.

> You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branch-network.adoc[Branch Network formula].

Reviewer: ```suggestion
You can disable this behaviour by toggling the [systemitem]``Do not append unique suffix to the salt client ID`` checkbox in the xref:salt:formula-branchnetwork.adoc[].
```
